src/main/java/core/basesyntax/GetFigure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class GetFigure implements GetFigureArea {
```

Comment:
this should be abstract class `Shape` or `Figure`. This is the base of your hierarchy - all your figures subclass it. 
Never name classes with verbs, only with nouns. No need to have all this logic, leave only color, and constructor for color.

Reply:
Create interface `Drawable` with void method `draw` and implement it here, then override in subclasses

Reply:
Except class name, I disagree with these comments. I don't understand what exactly is done wrong.